src/components/Card/Card.js

```diff
@@ -9,6 +9,8 @@ export const Card = ({ card, remove }) => {
           <p className='card-quote'>{card.quote}</p>
         </div>
         <img src={card.img}/>
+        <div className='share-btns'>
```

Comment:
Is this share button being used? If not, maybe remove it for now.

Reply:
Good eye. Thank you!